packagingenv/centos8/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM centos:8
+
+RUN yum -y install gcc-c++ make git libicu-devel rpmdevtools
+
+# Add NodeSource repo
+RUN curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
+
+# Install development tools
+RUN dnf -y module install nodejs:10
+
+# Install python3 for gyp
+RUN yum -y install python3 python36-rpm-macros
```

```suggestion
RUN yum -y install python3
```

Looks like `python36-rpm-macros` could be removed per previous https://github.com/StackStorm/st2chatops/pull/140#discussion_r368946602 comment.

---

we need this for the `%undefine __brp_mangle_shebangs` function

---

See https://github.com/StackStorm/st2chatops/pull/140#discussion_r368946602

---

I think we do need this in the build container

---

What's the detailed story, was the build failing if we remove that dependency?

That's the info I have according to https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-python3_configuring-basic-system-settings

> The python36-rpm-macros package is required so that files with /usr/bin/python3 shebangs are automatically changed to /usr/bin/python3.6. For more information, see Section 8.4.4, “Handling hashbangs in Python scripts”.

---

ok. I just submitted it with out it. Stuck behind st2 build. Gotta run for now.

---

Thanks :+1: